core/src/main/java/io/aiven/klaw/repository/KwKafkaConnectorRequestsRepo.java

```diff
@@ -31,9 +31,9 @@ boolean existsByTenantIdAndEnvironmentAndRequestStatusAndRequestOperationTypeAnd
 
   @Query(
       value =
-          "select count(*) from kwkafkaconnectorrequests where env = :envId and tenantid = :tenantId and connectorstatus='created'",
+          "select exists(select 1 from kwkafkaconnectorrequests where env = :envId and tenantid = :tenantId and connectorstatus='created')",
       nativeQuery = true)
-  List<Object[]> findAllConnectorRequestsCountForEnv(
```

Comment:
Can we change this one to Jpql?

Reply:
the problem is that there is a condition `connectorstatus='created'`
do you know how we can do it in jpql?

Reply:
Yeah should be like this.

boolean existsByTenantIdAndEnvironmentAndRequestStatus(
      int tenantId, String environment, String requestStatus);
      
      in SelectDataJDBC it goes to 
      kafkaConnectorRepo.existsByEnvironmentAndTenantId(env, tenantId, RequestStatus.CREATED.value);
      
      it will be reuseable for any status as well then!

Reply:
Actually your comment here might have just highlighted a bug on how we do it for schemaRequests because we aren't looking for the created status and we should be. I'm just going to take a quick look.

Reply:
to be honest this is not jpql... i was curius whether it's possible to do it with pure jpql...

otherwise i would prefer to do it on interface level to have both methods in same place

Reply:
ok we can leave it as is

Reply:
i've already applied changes
so now it's not `native`